data/sql/updates/pending_db_world/rev_1680298149569301100.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Harbinger Skyriss Normal Loot
+UPDATE `reference_loot_template` SET `GroupId`='3' WHERE `Entry`=25004 AND `Item` IN (28413, 28414, 28415, 28416, 28418, 28419);
```

Comment:
quotes around integers
```suggestion
UPDATE `reference_loot_template` SET `GroupId`=3 WHERE `Entry`=25004 AND `Item` IN (28413, 28414, 28415, 28416, 28418, 28419);
```

Reply:
Quotes required for swag,